libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -127,8 +127,21 @@ class ClientContext
         }
         _connect_pending = 1;
         _op_start_time = millis();
+        /*LEA*/
+        //Serial.print("\nClientContext::connect (A)");
+        uint32_t    nextPrint = (_op_start_time + 100);
+        while ((millis() < (_op_start_time + _timeout_ms)) &&
```

Comment:
Can't do millis() compare like this.  Need to subtract now from the start millis and compare to the timeout value.  OTW you'll end up with bad things when millis or millis + timeout rolls over.

Reply:
Ohhh... the ClientContext changes wasn‘t supposed to be part of this... sorry!